core/base_contract.go

```diff
@@ -207,7 +211,11 @@ func (bc *BaseContract) TxHealthCheck(_ *types.Sender) error {
 	return nil
 }
 
-func (bc *BaseContract) GetID() string {
+func (bc *BaseContract) ID() string {
+	return bc.GetID()
+}
+
+func (bc *BaseContract) GetID() string { // deprecated
 	return bc.config.GetSymbol()
 }
```

Comment:
Why don't you do it the other way around? What's deprecated, run through what's new?

Reply:
Good point!